controllers/welcomeController.js

```diff
@@ -1,11 +1,11 @@
-import dotenv from "dotenv";
-import db from "../models/databaseUrl";
+import dotenv from 'dotenv';
+
 dotenv.config();
 
 class WelcomeController {
   static welcome(req, res) {
     res.status(200).json({
-      msg: "Welcome to Authors Haven API, a #92Explorers Product."
```

Comment:
since you wrote this class, I prefer you please use the full form of the word message